rubric/rubric_writing.md

```diff
@@ -13,11 +13,15 @@ This rubric measures the student's ability to produce an assignment that *is wel
   </tr>
   <tr>
 	<td>Not Attempted</td>
-	  <td>Meaning of text is obscured due to significant grammar and spelling errors. <br></br> The submission is far too long (or far too short); word or sentence count significantly exceeded if one was given.</td>
-	<td>Many (> 10) grammatical and/or spelling errors are present but the meaning of text is not significantly obscured by grammar or spelling errors.</td>
-	<td>Fewer than 10 grammatical or spelling errors are present.</td>
-	<td>Fewer than 5 grammatical or spelling errors are present.  <br></br> The submisison is not too long or too short; if there was a word or sentence count given, then it was not exceeded by any significant margin.</td>
-	<td>Fewer than 1-2 grammatical or spelling errors are present.<br></br>  The submission is concise and to the point; the word or sentence count was respected.</td>
+	  <td>Meaning of text is obscured due to writing related issues such as significant grammar and spelling errors. <br></br> The submission is far too long (or far too short); word or sentence count significantly exceeded if one was given.</td>
+    <td>Many grammatical and/or spelling errors are present (> 10/250 words), but the meaning of text is not significantly obscured by grammar or spelling errors.</td>
+<br/><br/> The structure of the text is confusing and the reader may have to jump back and forth through the text. Any figures included are not labelled clearly or sensibly.
```

Comment:
Just wondering, why not placing the structuring descriptions for each grade range also in the table?

Reply:
Formatting error, thanks for catching it!